src/modules/commander/HealthAndArmingChecks/checks/cpuResourceCheck.hpp

```diff
@@ -34,10 +34,19 @@
 #pragma once
 
 #include "../Common.hpp"
+#include <lib/mathlib/math/filter/AlphaFilter.hpp>
 
 #include <uORB/Subscription.hpp>
 #include <uORB/topics/cpuload.h>
 
+using namespace math;
+using namespace time_literals;
+
+// Time constant for the alpha filter for averaging out the Cpu load over time
+// Currently, `cpu_load` message is being published at 2Hz, so assume 3-sample
+// timewindow as the alphafilter time constant
+static constexpr float CPULOAD_ALPHAFILTER_TIME_CONSTANT = 1.5f;
```

Comment:
if you want to tune it as a simple moving average filter, use this instead: https://en.wikipedia.org/wiki/Moving_average#Relationship_between_SMA_and_EMA
and simply set alpha in the constructor